ethexe/ethereum/src/lib.rs

```diff
@@ -154,6 +154,7 @@ impl Ethereum {
                     _mirror: mirror_address,
                     _mirrorProxy: mirror_proxy_address,
                     _wrappedVara: wvara_address,
+                    _middleware: Address::default(),
```

Comment:
this place should be the same as `ethexe/contracts/script/Deployment.s.sol`

Reply:
I think you can do it in the next PR such as #4403 for the Rust part, and do the Solidity part in this PR